packages/core/mesh/network-manager/src/swarm/peer.ts

```diff
@@ -108,6 +108,10 @@ export class Peer {
   async onOffer(message: OfferMessage): Promise<Answer> {
     const remoteId = message.author;
 
+    if (this.connection && this.connection.state !== ConnectionState.INITIAL) {
```

Comment:
if we ever "bring back" connection limiter, then connection's first state is actually `CREATED`, which gets switched to `INITIAL` after it went through the limiter, so maybe it should be something like
```
if (this.connection && ![ConnectionState.CREATED, ConnectionState.INITIAL].includes(this.connection.state)) {
```